content/articles/building-a-simple-translation-app-using-python-for-beginners/index.md

```diff
@@ -0,0 +1,151 @@
+Want to build your very own language translation app?
+
+Well, in this tutorial, I'll show you how you can build one yourself using Hugging Face Transformers and Gradio!
+
+### Prerequisite
+
+To understand the contents of this tutorial, a reader needs to be familiar with the Python programming language and machine learning. I'm also going to be using [Google Colab](https://colab.research.google.com) notebook. You need to be familiar with that too.
+
+### Table of contents
+
+1. [Introduction](#introduction)
+2. [Installing dependencies](#installing-dependencies)
+3. [Downloading the pre-trained translation model](#downloading-the-pre-trained-translation-model)
+4. [Building the web app for interaction using Gradio](#building-the-web-app-for-interaction-using-gradio)
+5. [Wrapping up](#wrapping-up)
+
+### Introduction
+
+[Hugging Face](https://huggingface.co/transformers/) is a great open-source library that is doing powerful work in the Natural Language Processing (NLP) space. The library has a bunch of pre-trained models which you can leverage or fine-tune. Rather than going ahead and training a huge language model, one can leverage this library instead.     
+
+[Gradio](https://gradio.app/), on the other hand, allows us to quickly create customizable UI components around PyTorch models.
+
+### Installing dependencies
+
```

Comment:
This section can be a bit confusing to the reader as you've mentioned at the end of the section which ones to install and which ones not to. If you could shift that to the beginning of the paragraph and ensure that a reader would be able to grasp which code blocks to run on his machine and which code blocks to run on the Colab notebook, it'd be much better.